glossary.md

```diff
@@ -1,16 +1,84 @@
 You may also need to check https://chromium.googlesource.com/chromiumos/docs/+/master/glossary.md.
 
-* LGTM: "Looks good to me", commonly used to approve a code review.
-* PTAL: Please take a look.
-* RSLGTM: "Rubber-stamp looks good to me". The reviewer approving without doing
+* ABI: Application Binary Interface - contracts for internal
```

Comment:
Do we really need entries that can be easily looked up in Wikipedia or OS textbooks and widely used in the software world in general? The same goes to things like CLI, EOF, ELO, RAII, etc.

Reply:
@joyeecheung, (a) the problem this gloassry addresses is difficulty in discovering meaning of abbreveations, and solution is to keep those in one place for easy reference; based on our usage of those, not based on how trivial those are? (b) I did a grep in the code base and found repeated usage of these, (c) these items were proposed in the original issue by maintainers